src/addons/FluxComponent.js

```diff
@@ -75,7 +75,6 @@ class FluxComponent extends React.Component {
       render,
       connectToStores,
       stateGetter,
```

Comment:
these all can actually be removed too, unless you're defining them here for documentation purposes.